src/main/java/com/alkemy/ong/auth/config/SecurityConfiguration.java

```diff
@@ -49,8 +49,8 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
         httpSecurity.csrf().disable()
                 .authorizeRequests().antMatchers("/auth/*").permitAll()
                 .antMatchers("/auth/register/**").hasAnyAuthority("ADMIN", "USER")
-                .antMatchers(GET, "/Users/**").hasAnyAuthority("ROLE_USER")
-                .antMatchers(POST, "/User/save/**").hasAnyAuthority("ROLE_ADMIN")
+                .antMatchers(GET, "/users/**","/users/list/**").hasAnyRole("ROLE_ADMIN")
```

Comment:
this line blocks all ROLE_USERs from using the getUsers routes, is that correct?

Reply:
I forgot to remove /users/**, thanks for your observation